### PR TITLE
more Doxygen updates

### PR DESCRIPTION
**Pull request by jbathegit**
Yes, it's a lot of files, but everything is documentation updates **except** for moda_ifopbf.F being renamed and updated to modv_IFOPBF.f90

I'm going to continue to plug away at a little more documentation, and then by the middle of next week I expect to be ready to do a release as discussed in #131 and in https://github.com/NOAA-EMC/hpc-stack/issues/199

### CODE REVIEW

File: src/ardllocc.c [hunk 1]
```diff
@@ -1,42 +1,28 @@
 /** @file
-    @author ATOR @date 2014-12-04
```
Comment (edwardhartnett) [inline src/ardllocc.c:2]: I would perfer if you left the author tag at all levels - file, subprogram, and module. Each are different (in general).

I realize in your code you have one subprogram per file, but it needed stay that way. You could will the lottery, move to Tahiti, and then your replacement might decide to merge many subprograms in one file. The file authorship will then be different.

In general, as software libraries like this are maintained over the years and decades, this author information will no longer seem so redundant. Code arrangement drifts over time...

File: src/ardllocf.F [hunk 2]
```diff
@@ -1,40 +1,18 @@
...
-C>   SUBROUTINE SHOULD ONLY BE CALLED DIRECTLY BY AN APPLICATION PROGRAM
-C>   IF THE PROGRAM IS COMPLETELY FINISHED WITH ALL CALLS TO ALL OTHER
-C>   BUFR ARCHIVE LIBRARY ROUTINES, BECAUSE THE MEMORY FREED HEREIN WILL
-C>   RENDER THE LIBRARY AS EFFECTIVELY UNUSABLE FOR THE REMAINDER OF THE
-C>   LIFE OF THE APPLICATION PROGRAM.  HOWEVER, THIS MAY BE A USEFUL
-C>   OPTION FOR APPLICATION PROGRAMS WHICH WANT TO MOVE ON TO OTHER
-C>   UNRELATED TASKS WITHOUT CONTINUING TO TIE UP A SIGNIFICANT AMOUNT
-C>   OF DYNAMICALLY-ALLOCATED HEAP MEMORY RELATED TO THIS LIBRARY.
-C>   OTHERWISE, ALL SUCH MEMORY WILL BE FREED AUTOMATICALLY ONCE THE
-C>   APPLICATION PROGRAM TERMINATES.
```
Comment (edwardhartnett) [inline src/ardllocf.F:21]: Seems like you lost a lot of the text here. Was that intentional?

File: src/bort.f [hunk 3]
```diff
@@ -1,75 +1,27 @@
...
-C>                               UFBMEX   UFBMMS   UFBMNS   UFBOVR
-C>                               UFBPOS   UFBQCD   UFBQCP   UFBREP
-C>                               UFBRMS   UFBSEQ   UFBSTP   UFBTAB
-C>                               UFBTAM   UFDUMP   UPDS3    UPFTBV
-C>                               UPTDD    USRTPL   WRCMPS   WRDESC
-C>                               WRDLEN   WRDXTB   WRITDX   WRITLC
-C>                               WRITSA   WRITSB   WTSTAT   CODFLG
-C>                               RDMTBF   INITTBF  STRTBFE  GETCFMNG
-C>                               Normally not called by any application
-C>                               programs but it could be.
```
Comment (edwardhartnett) [inline src/bort.f:67]: Great that we can now rely on doxygen to generate this list. I would not really believe it was correct, otherwise...

**Review by edwardhartnett — APPROVED** [review]

File: src/ardllocf.F (hunk 2 above)
Comment (jbathegit) [reply src/ardllocf.F:21]: Yes, this was intentional.  That text is now part of the docblock for exitbufr.f, which calls ardllocf.F internally.   So the information is still there, but just at a higher level in the hierarchy where the user is more likely to see it ;-)

**Pull request merged**